src/Features/LanguageServer/Protocol/Handler/Formatting/FormatDocumentHandlerBase.cs

```diff
@@ -13,28 +14,33 @@
 
 namespace Microsoft.CodeAnalysis.LanguageServer.Handler
 {
-    internal class FormatDocumentHandlerBase
+    internal abstract class FormatDocumentHandlerBase
```

Comment:
We're confident nobody was creating this directly?

Reply:
yes, no one outside of Roslyn has IVTs to this project